data/hai/hai reveal 2 start.txt

```diff
@@ -246,10 +246,10 @@ mission "Hai Rescue: Ooonem"
 			`	Following reports of the last time the 'angel of death' was sighted, you end up loitering in a twisting street parallel to a main thoroughfare. The few people using it hurry by quickly, and rarely seem to travel any great distance down this street. One very surly looking individual gives you a sour glare on his way past, a collection of weapons on full display. He looks a lot like you'd expect a bounty hunter to look, and as he moves out of sight you observe a shadow, previously motionless, detach from a spot on a wall and head back your direction, laser rifle pointed in the direction of the departing individual. As the shadow approaches, you see there's a lot of hair sticking out from their hood. Seems you may have indeed found a Hai. Do you want to talk to them?`
 			label start
 			choice
-				`	(Talk to her.)`
+				`	(Talk to them.)`
 				`	(Come back later.)`
 					defer
-			`	As step into her path, she swings the laser rifle towards you and says, "You're either very brave, very stupid, or know something... I should warn you, I'm not going back to them!"`
+			`	As you step into her path, the Hai swings the laser rifle towards you and says, "You're either very brave, very stupid, or know something... I should warn you, I'm not going back to them!"`
```

Comment:
`As you step into *her* path,`

Do we know her gender at this point or not?
Also, the next line says `she`. Do we know it by that point?

Reply:
Yes?

Reply:
It's softly implied by the construction of this that it's apparent once she speaks to you, but doesn't bear remarking upon.

Reply:
You've changed the earlier reference to be gender neutral, and a reference after the part I quoted, (though, maybe that was just to remove hte pronoun altogether?) and she doesn't start speaking until the end of this line.

Reply:
I changed `As you step into her path` to `As you step into their path` to maintain consistency.